examples/preprocessing/plot_resample.py

```diff
@@ -61,7 +61,7 @@
 # When resampling epochs is unwanted or impossible, for example when the data
 # doesn't fit into memory or your analysis pipeline doesn't involve epochs at
 # all, the alternative approach is to resample the continuous data. This
-# can also be done on non-preloaded data.
+# can only be done on loaded or pre-loaded data.
```

Comment:
None in practice, as we have `preload=True` and `inst.load_data()` that use different naming schemes to effectively accomplish the same things. Can you think of a clearer wording? Maybe "on data that has been loaded from disk into memory or a memmapped file, e.g. using `preload` keyword argument or `inst.load_data()`.", but that's a bit more verbose.